scripts/lst-sensitivity.py

```diff
@@ -13,51 +13,54 @@
 
 parser = argparse.ArgumentParser(description="Compute Sensitivity Curve.")
 
-parser.add_argument('--gammadl1', '-gd1', type=str,
-                    dest='dl1file_gammas',
-                    help='path to gammas simtelfile')
-parser.add_argument('--protondl1', '-pd1', type=str,
-                    dest='dl1file_protons',
-                    help='path to protons simtelfile')
-parser.add_argument('--gammadl2-cuts', '-gd2-cuts', type=str,
-                    dest='dl2_file_g_cuts',
-                    help='path to reconstructed gammas dl2 file')
-parser.add_argument('--protondl2-cuts', '-pd2-cuts', type=str,
-                    dest='dl2_file_p_cuts',
-                    help='path to reconstructed protons dl2 file')
-parser.add_argument('--gammadl2-sens', '-gd2-sens', type=str,
-                    dest='dl2_file_g_sens',
-                    help='path to reconstructed gammas dl2 file')
-parser.add_argument('--protondl2-sens', '-pd2-sens', type=str,
-                    dest='dl2_file_p_sens',
-                    help='path to reconstructed protons dl2 file')
+parser.add_argument('--gammadl1', '-gd1', type = str,
+                    dest = 'dl1file_gammas',
+                    help = 'path to gammas simtelfile')
+parser.add_argument('--protondl1', '-pd1', type = str,
+                    dest = 'dl1file_protons',
+                    help = 'path to protons simtelfile')
+parser.add_argument('--gammadl2-cuts', '-gd2-cuts', type = str,
+                    dest = 'dl2_file_g_cuts',
+                    help = 'path to reconstructed gammas dl2 file')
+parser.add_argument('--protondl2-cuts', '-pd2-cuts', type = str,
+                    dest = 'dl2_file_p_cuts',
+                    help = 'path to reconstructed protons dl2 file')
+parser.add_argument('--gammadl2-sens', '-gd2-sens', type = str,
+                    dest = 'dl2_file_g_sens',
+                    help = 'path to reconstructed gammas dl2 file')
+parser.add_argument('--protondl2-sens', '-pd2-sens', type = str,
+                    dest = 'dl2_file_p_sens',
+                    help = 'path to reconstructed protons dl2 file')
 
 args = parser.parse_args()
 
-nfiles_gammas = 0.5#100*0.5 #Pointlike gammas
-nfiles_protons = 0.5#5000*0.8*0.5
 
-eb = 20 # Number of energy bins
-gb = 11 #Number of gammaness bins
-tb = 10 #Number of theta2 bins
+ntelescopes_gamma = 4
+ntelescopes_protons = 4
+
+eb = 20  #  Number of energy bins
```

Comment:
explicit variables names (and then no need to comment what they mean)

Reply:
done